composer.json

```diff
@@ -1,49 +1,59 @@
 {
-    "name": "theofidry/psysh-bundle",
-    "type": "symfony-bundle",
+    "name": "teamdotblue/psysh-bundle",
     "description": "A port of the PHP REPL PsySH for Symfony.",
-    "keywords": ["console", "psysh", "shell", "REPL", "symfony", "interactive"],
     "license": "MIT",
-    "authors": [
-        {
-            "name": "Adrian Palmer",
-            "email": "navitronic@gmail.com"
-        },
-        {
-            "name": "Théo FIDRY",
-            "email": "theo.fidry@gmail.com",
-            "homepage": "https://github.com/theofidry"
-        }
+    "type": "symfony-bundle",
+    "keywords": [
+        "console",
+        "psysh",
+        "shell",
+        "REPL",
+        "symfony",
+        "interactive"
     ],
-
     "require": {
-        "php": ">=8.0",
-        "psy/psysh": "^0.11",
-        "symfony/error-handler": "^5.4|^6.0",
-        "symfony/expression-language": "^5.4|^6.0",
-        "symfony/framework-bundle": "^5.4|^6.0"
+        "php": ">=8.2",
+        "psy/psysh": "^0.11 || ^0.12",
+        "symfony/error-handler": "^6.4 || ^7.0",
+        "symfony/expression-language": "^6.4 || ^7.0",
+        "symfony/framework-bundle": "^6.4 || ^7.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^9.5",
-        "symfony/symfony": "^5.4|^6.0"
+        "ergebnis/composer-normalize": "^2.8",
+        "friendsofphp/php-cs-fixer": "^3.0",
+        "phpstan/phpstan": "^1.0",
+        "phpunit/phpunit": "^10.5",
+        "rector/rector": "^1.0"
     },
-
     "autoload": {
         "psr-4": {
-            "Fidry\\PsyshBundle\\": "src"
+            "TeamBlue\\PsyshBundle\\": "src"
```

Comment:
General conventions regarding composer packages want us to re-use the github username, so please use `TeamDotBlue\\PsyshBundle` instead?